frontend/actions/src/main/kotlin/com/virtuslab/gitmachete/frontend/actions/dialogs/GitNewCommitMessageActionDialog.kt

```diff
@@ -0,0 +1,62 @@
+package com.virtuslab.gitmachete.frontend.actions.dialogs
+
+import com.intellij.openapi.editor.impl.EditorImpl
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.ui.DialogWrapper
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.vcs.ui.CommitMessage
+import com.intellij.ui.components.JBLabel
+import com.intellij.util.ui.JBUI
+import com.intellij.util.ui.UIUtil
+
+internal class GitNewCommitMessageActionDialog(
+    val project: Project,
+    val originMessage: String,
+    title: String,
+    val dialogLabel: String
+) : DialogWrapper(project, true) {
+  val commitEditor = createCommitEditor()
+  var onOk: (String) -> Unit = {}
+
+  init {
+    Disposer.register(disposable, commitEditor)
+
+    init()
+    isModal = false
+    this.title = title
+  }
+
+  fun show(onOk: (newMessage: String) -> Unit) {
+    this.onOk = onOk
+    show()
+  }
+
+  override fun createCenterPanel() =
+      JBUI.Panels.simplePanel(UIUtil.DEFAULT_HGAP, UIUtil.DEFAULT_VGAP)
+          .addToTop(JBLabel(dialogLabel))
+          .addToCenter(commitEditor)
+
+  override fun getPreferredFocusedComponent() = commitEditor.editorField
+
+  override fun getDimensionServiceKey() = "Git.Rebase.Log.Action.NewCommitMessage.Dialog"
```

Comment:
Huh where does this constant come from? maybe can/should be imported instead?

Reply:
good point. I do not remember y I did copypasted that. it works as import